web/client/api/GeoStoreDAO.js

```diff
@@ -29,6 +29,10 @@ let parseUserGroups = (groupsObj) => {
     return groupsObj.User.groups.group.filter(obj => !!obj.id).map((obj) => _.pick(obj, ["id", "groupName", "description"]));
 };
 
+const encodeContent = function(content) {
+    return jsesc(content, {json: true, wrap: false, quotes: 'backtick'});
```

Comment:
So the backtick will be escaped? Is there a way to disable quotes escaping totally ?

Reply:
Not with jsesc (unless we patch it). The options are:
 * forking and making a PR to the project
 * reimplementing the escaping code only for unicode chars, without using this lib

Reply:
So if a user insert a backquote in a resource for instance, to break everything. We are planning to use markdown somewere for rich text, where backquote is part of the syntax.
I still didn't get why we have to use that lib. Did we do all the checks, as explained here? 
https://www.w3.org/International/questions/qa-changing-encoding

Reply:
Yes, it seems to be working only if we encode unicode chars with hex sequences. jsesc is doing this, but also does more and you cannot disable everything (like quote encoding). As I said, we can:
 * implement a flag in jsesc to only do unicode chars encoding
 * implement in MS2 a piece of code doing only that

Reply:
This pull request moves the problem to a less used char. I think we should solve the problem at all.